read_utils.py

```diff
@@ -92,16 +94,6 @@ def fastq_to_fasta(inFastq, outFasta):
     return 0
 
 
-def parser_fastq_to_fasta(parser=argparse.ArgumentParser()):
```

Comment:
I doubt anyone is depending on this, but any reason not to leave it exposed by the CLI?

Reply:
This old bit of code seemed to serve one internal purpose, but wasn't really providing any utility for user from a higher level point of view. I don't think we need to provide a comprehensive set of general utils from biopython / pysam -- just common operations for our own use. I have a branch going that I intend to PR later on that actually eliminates this function altogether.

Reply:
Got it. Always nice to remove stale bits.